packages/uploadthing/src/internal/types.ts

```diff
@@ -48,31 +41,26 @@ type ResolverOptions<TParams extends AnyParams> = {
 };
 
 export type AnyRuntime = "app" | "pages" | "web" | "express" | "fastify";
+
+export type MiddlewareFnArgs<TRequest, TResponse> = {
+  req: TRequest;
+  res: TResponse;
```

Comment:
will extend this to also include an `event` prop [to support H3](https://github.com/pingdotgg/uploadthing/pull/358). 

That was actually the rationale for this PR in the first place :) https://github.com/pingdotgg/uploadthing/actions/runs/6206560816/job/16850938399?pr=358#step:8:512